common/color_definitions.scss

```diff
@@ -1,6 +1,6 @@
-$accent-color: dark-light-choose(#3c41c3, #6e4bbc);
+// $accent-color: dark-light-choose(#3c41c3, #6e4bbc);
```

Comment:
We really should just delete files instead of commenting them, we can always get them back later from git